### PR TITLE
Clone edx-themes and mount for whitelabel configuration.

### PR DESCRIPTION
**Pull request by doctoryes**
https://openedx.atlassian.net/browse/PLAT-1975
This PR adds code to clone edx-themes (separately from the usual cloning) and adds volume mount points for edx-themes in the appropriate containers.

### CODE REVIEW

File: repo.sh [hunk 1]
```diff
@@ -27,11 +27,19 @@ repos=(
     "https://github.com/edx/xqueue.git"
 )
 
+private_repos=(
+    # Needed to run whitelabel tests.
+    "https://github.com/edx/edx-themes.git"
```
Comment (doctoryes) [inline repo.sh:32]: Is it OK to name a private GitHub repo here?

**Review by coryleeio — APPROVED** [review]

File: repo.sh (hunk 1 above)
Comment (bmedx) [reply repo.sh:32]: Seems ok to me, but I think we need further documentation about why this won't ever work for people outside of the org. Maybe in the Makefile comments and devstack README?

**Review by bmedx — APPROVED** [review]

File: repo.sh (hunk 1 above)
Comment (doctoryes) [reply repo.sh:32]: Yeah - I think what I'll do is just split the make targets which clone these into a separate edx-specific Makefile. So then it'd be run with:
`make -f Makefile.edx dev.clone_whitelabel`

**Comment by doctoryes** [comment]
@bmedx What do you think about the edx-specific Makefile I've added?

**Comment by bmedx** [comment]
Looks good! Much clearer, I think.

**Pull request merged**